node/worker/worker.go

```diff
@@ -492,15 +485,39 @@ func (w *Worker) FinalizeNewBlock(
 	}
 	state := w.current.state.Copy()
 	copyHeader := types.CopyHeader(w.current.header)
```

Comment:
is there a side-effect to not copying the sig and bitmap to `w.current.header`? earlier, everything was written to `w.current.header` and then deep-copied.

Reply:
it is fine since the commit sigs are empty is the header here. Will be populated later.